main.cpp

```diff
@@ -127,393 +163,337 @@ struct CarWash
-    // width in inches
     double width = 48.0;
-    // composition material
     std::string compositionMaterial = "glass";
+    std::string drawerLocation = "in";
 
-    // Slide keyboard drawer
     std::string slideDrawer();
-    // Adjust height
-    float changeHeight(float heightChange = 1.f);
-    // roll to new location
+    void changeHeight(float heightChange = 1.f);
     float rollDesk(float xDistance = 0.5f, float yDistance = 0.5f);
-
 };
-/*
-Thing 3) Guitar amplifier
-5 properties:
-    1) Preamplifier gain (double)
-    2) number of channels (int)
-    3) max output power in watts (float)
-    4) treble gain (double)
-    5) reverb amount (double)
-3 things it can do:
-    1) amplify guitar input
-    2) adjust reverb amount
-    3) switch channel
- */
+
+std::string StandingDesk::slideDrawer()
+{ 
+    if (drawerLocation == "in") return "out";
+
+    return "in";
+}
+
+void StandingDesk::changeHeight(float heightChange)
+{
+    deskHeight += heightChange;
+}
+
+float StandingDesk::rollDesk(float xDistance , float yDistance)
+{
+    return xDistance + yDistance;
+}
+
 struct GuitarAmp
 {
-    //  Preamplifier gain
     double preampGain = 0.25;
-    //  number of channels
     int numChannels = 2;
-    // max output power in watts
     float maxOutPower = 100.f;
-    // treble gain
     double trebleGain = 0.5;
-    // reverb amount
     double reverbLevel = 0.25;
 
     struct PowerAmp
     {
-        // Amp gain
         double gain = 0.0;
-        // Power amp config
         std::string ampTopology = "Class AB";
-        // output impedance in ohms
         int outputImpedance = 8;
-        // number of outputs
         int numOutputs = 1;
-        // power state
         std::string powerState = "standby";
 
-
-        // Change gain
-        void changeGain(double gainAdjustment = 1.0);
-        // Put amp into standby
+        void changeGain(double gainAdjustment);
         void enableAmpStandby();
-        // select new output impedance
-        int selectImpedance( int impedance = 8);
+        int selectImpedance( int impedance = 8) ;
     };
 
-    // amplify guitar input
+
     double amplifyGuitar(double inputVoltage = 0.0, double ampGain = 0.0);
-    // adjust reverb amount
-    double adjustReverbAmount(double reverbLevel = 0.25, double reverbGain = 1.0);
-    // switch channel
+    double adjustReverbAmount(double reverbGain = 1.0);
     int switchChannel(int currentChannel, int newChannel = 0);
 };
-/*
-Thing 4) Power strip
-5 properties:
-    1) Number of power outlets (int)
-    2) max surge  protection rating in watts (float)
-    3) connector format (std::string) 
-    4) wire gauge (int)
-    5) number of populated outlets (int)
-3 things it can do:
-    1) Install plug
-    2) get current outlet power 
-    3) disable power
- */
+
+void GuitarAmp::PowerAmp::changeGain(double gainAdjustment) {gain *= gainAdjustment;}
+void GuitarAmp::PowerAmp::enableAmpStandby() { powerState = "standby";}
```

Comment:
If you write single-line functions, make sure you have space around the `{ }` so the code is easier to read
- [ ] revise